src/main/java/io/github/wimdeblauwe/ttcli/Init.java

```diff
@@ -59,6 +59,7 @@ public void init(@ShellOption(defaultValue = ".", value = "baseDir") String base
             ComponentFlow.ComponentFlowResult flowResult = flow.run();
 
             ComponentContext<?> context = flowResult.getContext();
+            String type = context.get("type");
```

Comment:
Maybe use `String projectType = context.get("project-type");` here as just `type` is a bit unclear.